Dockerfile

```diff
@@ -29,4 +31,9 @@ RUN apt-get -yqq update && apt-get -yqq install \
 
 ENV FWROOT=/FrameworkBenchmarks PYTHONPATH=/FrameworkBenchmarks
 
+# Drop permissions of user to match those of the host system
+RUN addgroup --gid $GROUP_ID user
```

Comment:
You should add the group and the user only if they do not exist already - for example, on my macOS `13.1 (22C65)` system with Docker Desktop version `4.17.0 (99724)` this fails with the following error:
```
[+] Building 1.5s (6/7)                                                                                                                                                                                                                                                       
 => [internal] load build definition from Dockerfile                                                                                                                                                                                                                     0.0s
 => => transferring dockerfile: 1.40kB                                                                                                                                                                                                                                   0.0s
 => [internal] load .dockerignore                                                                                                                                                                                                                                        0.0s
 => => transferring context: 2B                                                                                                                                                                                                                                          0.0s
 => [internal] load metadata for docker.io/library/ubuntu:18.04                                                                                                                                                                                                          1.2s
 => [1/4] FROM docker.io/library/ubuntu:18.04@sha256:1e32b9c52e8f22769df41e8f61066c77b2b35b0a423c4161c0e48eca2fd24f75                                                                                                                                                    0.0s
 => CACHED [2/4] RUN apt-get -yqq update && apt-get -yqq install       -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold"       cloc       dstat                       `# Collect resource usage statistics`       git-core       libmysqlclient  0.0s
 => ERROR [3/4] RUN addgroup --gid 20 user                                                                                                                                                                                                                               0.2s
------                                                                                                                                                                                                                                                                        
 > [3/4] RUN addgroup --gid 20 user:
#6 0.222 addgroup: The GID `20' is already in use.
------
executor failed running [/bin/sh -c addgroup --gid $GROUP_ID user]: exit code: 1
Unable to find image 'techempower/tfb:latest' locally
docker: Error response from daemon: pull access denied for techempower/tfb, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
```
FYI `id -u` and `id -g` return `501` and `20` respectively in my environment.

Reply:
I've switched this to use a check first